### ISSUE TITLE
Standardize spelling and hyphenation of tree-shakable

### ISSUE THREAD

**Issue by jenniferfell** [angular/angular#23822]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[X ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
[ ] Other... Please describe:
</code></pre>

## Current behavior

Another question, is it Tree-Shakable or Tree-Shakeable?
Both word have occurrence in doc, codebase and issues:

    https://github.com/angular/angular/search?utf8=%E2%9C%93&q=tree-shakable&type=
    https://github.com/angular/angular/search?utf8=%E2%9C%93&q=tree-shakeable&type=

## Expected behavior

Original request from @trotyl : As a non-English speaker I'm not sure which one is better, but strongly believe only one should be used.

## Environment

<pre><code>
Angular version: X.Y.Z

Browser:
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

**Comment by jenniferfell** [comment]
Decision: tree-shakable

Why: While both spellings are acceptable according to different sources, we had to pick one. 
- Google dev docs standardize on US English (https://developers.google.com/style/translation).
- US English tends toward dropping the "e" in similar words (usable, customizable, executable).
- Mirriam-Webster (a US standard dictionary) does not recognize "shakeable." 

We recognize that UK English tends to keeps the "e" in words like sizeable, shareable, and presumably also shakeable. In recent years, US English and UK English are both becoming more flexible about the spelling of these words. Google's search engine matches on both spellings. Still, we had to pick one. 

Thanks for the nudge, @trotyl  :+1:
